src/main/plugin/iso19139.ca.HNAP/loc/fre/labels.xml

```diff
@@ -2422,9 +2422,13 @@
       <option value="HTTP">HTTP</option>
       <option value="HTTPS">HTTPS</option>
       <option value="FTP">FTP</option>
-      <option value="ESRI REST: Map Service">ESRI REST: Map Service</option>
+      <option value="ESRI REST: Map Service">ESRI REST: service de carte</option>
       <option value="OGC:WMS">OGC:WMS</option>
       <option value="OGC:WFS">OGC:WFS</option>
+      <option value="OGC API – Coverages">API OGC – Couvertures</option>
+      <option value="OGC API – Features">API OGC – Fonctionnalités</option>
```

Comment:
I don't feel that Features translates well to Fonctionnalités

According to ArcGIS
https://pro.arcgis.com/fr/pro-app/latest/help/data/services/use-ogc-api-services.htm
OGC API Features (Entités d’API de l’OGC)

So "features" are translated to "entités" which seems better.

So you could use "OGC API - Entités"

Reply:
Updated